### PR TITLE
uhubctl 2.6.0 (new formula)

### PR DESCRIPTION
**Pull request by mvp**
Adding new formula for [uhubctl](https://github.com/mvp/uhubctl) utility (it has over 2000 stars and over 200 forks).

Validated that all tests and audit checks for new uhubctl formula are passing.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/u/uhubctl.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  desc "USB hub per-port power control"
+  homepage "https://github.com/mvp/uhubctl"
+  url "https://github.com/mvp/uhubctl/archive/refs/tags/v2.6.0.tar.gz"
+  sha256 "56ca15ddf96d39ab0bf8ee12d3daca13cea45af01bcd5a9732ffcc01664fdfa2"
+  license "GPL-2.0-only"
+  head "https://github.com/mvp/uhubctl.git", branch: "master"
+
+  livecheck do
+    url :stable
+  end
```
Comment (SMillerDev) [inline Formula/u/uhubctl.rb:11]: No need to add this, it is the default already

File: Formula/u/uhubctl.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  depends_on "pkgconf" => :build
+  depends_on "libusb"
+
+  def install
+    system "make"
+    bin.install "uhubctl"
+  end
+
+  test do
+    system bin/"uhubctl", "-v"
```
[the remark this answers is not in the document]
Comment (mvp) [reply Formula/u/uhubctl.rb:22]: uhubctl operates on actually connected USB devices to turn port power off or on. I can't come up with any functional test that would be of any use without having supported physical hub connected. Of course that is effectively impossible in CI or docker environment.

This test running with -v simply validates that generated executable is usable.

File: Formula/u/uhubctl.rb (hunk 1 above)
Comment (mvp) [reply Formula/u/uhubctl.rb:11]: Thanks, I will remove it

File: Formula/u/uhubctl.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (mvp) [reply Formula/u/uhubctl.rb:22]: Not sure what you mean by this. If there are no USB hubs connected, `uhubctl` invocation will print:
```
$ uhubctl
No compatible devices detected!
Run with -h to get usage info.
```
But this is not really any better than just `uhubctl -v`, is it?
Also, it will actually print something useful it test is running on host with USB hubs connected and negative test would fail.

Or, I can add this test for definitely non-existent device:
```
$ uhubctl -l 100-1.2.3.4.5 -a 0 -p 1
No compatible devices detected at location 100-1.2.3.4.5!
Run with -h to get usage info.
```
What do you think?

File: Formula/u/uhubctl.rb (hunk 1 above)
Comment (mvp) [reply Formula/u/uhubctl.rb]: removed

File: Formula/u/uhubctl.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (mvp) [reply Formula/u/uhubctl.rb]: Added negative test, please review

**Comment by mvp** [comment]
@SMillerDev: I added commit to address review feedback, but now getting error about commit style.
Do I need to force push with single amended commit? I saw that force push may confuse CI builder.
Thanks!

**Comment by daeho-ro** [comment]
@mvp Yes, the commit is needed to squash. Otherwise, all looks good.

**Comment by mvp** [comment]
> @mvp Yes, the commit is needed to squash. Otherwise, all looks good.

Thank you @daeho-ro, squashed and force pushed

**Review by daeho-ro — APPROVED** [review]
